apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingsstatusRoutes.kt

```diff
@@ -96,10 +96,12 @@ internal fun Route.behandlingsstatusRoutes(behandlingsstatusService: BehandlingS
     }
 }
 
+data class OperasjonGyldig(val gyldig: Boolean)
+
 private suspend fun haandterStatusEndring(call: ApplicationCall, proevStatusEndring: () -> Unit) {
     runCatching(proevStatusEndring)
         .fold(
-            onSuccess = { call.respond(HttpStatusCode.OK, "true") },
```

Comment:
Kan vera vi bør ha tilsvarande for alle `call.respond(HttpStatusCode.OK, "true")` i denne fila

Reply:
denne metoden brukes av alle endepunktene, eller tenkte du på noe annet?

Reply:
Oi, her var det eg som såg på ein gammal versjon av koden 😊